pkg/apis/garden/validation/validation_test.go

```diff
@@ -243,6 +260,18 @@ var _ = Describe("validation", func() {
 						"Field": Equal(fmt.Sprintf("spec.%s.constraints.kubernetes.offeredVersions[].expirationDate", fldPath)),
 					}))
 				})
+
+				It("should forbid duplicated kubernetes versions", func() {
+					awsCloudProfile.Spec.AWS.Constraints.Kubernetes.OfferedVersions = duplicatedKubernetes
+
+					errorList := ValidateCloudProfile(awsCloudProfile)
+
+					Expect(errorList).To(HaveLen(1))
+					Expect(*errorList[0]).To(MatchFields(IgnoreExtras, Fields{
```

Comment:
Use [`ConsistsOf`](https://github.com/gardener/gardener/blob/master/pkg/apis/garden/validation/validation_test.go#L120-L136) for all new tests

Reply:
Done, PTAL